js/src/lib/components/InferenceWidget/widgets/AudioClassificationWidget/AudioClassificationWidget.svelte

```diff
@@ -122,37 +122,36 @@
 		throw new TypeError("Invalid output: output must be of type Array<label: string, score:number>");
 	}
 
-	function applyInputSample(sample: WidgetExampleAssetInput<WidgetExampleOutputLabels>) {
-		file = null;
+	function applyInputSample(
+		sample: WidgetExampleAssetInput<WidgetExampleOutputLabels>,
+		{ isPreview = false, inferenceOpts = {} }: ExampleRunOpts = {}
```

Comment:
Would be clearer like this IMO:


```suggestion
		opts: ExampleRunOpts = {}
```

Reply:
agreed

Reply:
handled in https://github.com/huggingface/hub-docs/pull/1023/commits/7858186d98bada14f62fd0bb669eb894076d5276